### ISSUE TITLE
A11y_AzureDataStudio_New deployment - SQL Server on Windows_Screen Reader: Narrator is not announcing previous as button in Microsoft sql server license terms page

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18590]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info and Relationships; #Narrator; #NVDA; #Element:Button; 

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment"
5. In the Deployment options, select "SQL Server on Windows" card.
5. From the dropdown for Version under Options, select from the available options.
6. Click the "Select" button  
7. This will start the setup installer for SQL Server setup. 
8. Follow the instructions through the setup process to install a SQL Server on Windows.
9. Verify the previous button in Microsoft sql server license terms page is announcing by narrator as button or not

### Actual Result:
Narrator is not announcing previous as button in Microsoft sql server license terms page

### Expected Result:
Narrator should announce previous as button in Microsoft sql server license terms page

### Attachment:
**Narrator Attachment:**
![A11y_AzureDataStudio_New deployment - SQL Server on Windows_Screen Reader](https://user-images.githubusercontent.com/100189407/156147165-3e400d4b-c94e-4981-8ae3-2ee8fb6ff60c.png)

**NVDA Attachment:**
![A11y_AzureDataStudio_New deployment - SQL Server on Windows_Screen Reader_NVDA](https://user-images.githubusercontent.com/100189407/156147368-3fff235d-6158-41fa-9597-542073b8071c.png)

**Comment by alanrenmsft** [comment]
the install SQL server feature you tested is not a part of AzureDataStudio. so closing it, let me know if you don't know the owner of the sql server installer.

**Comment by VijayaBharathyA** [comment]
@alanrenmsft Could you please let us know if we need to route this bug to the SQL server installer team? If yes, could you please do let us know the vso and owner's details?

**Comment by VijayaBharathyA** [comment]
@alanrenmsft Could you please route it to the appropriate owner or team and reactivate the bug.
**Note:** "Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"

**Comment by VijayaBharathyA** [comment]
**Verified issue in below Environment:**
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Issue is fixed
![18590_Previous Button (1)](https://user-images.githubusercontent.com/100189407/216270605-05b0d973-03f8-4553-b6b4-3e4dd89b416e.gif)
